lib/linguist/languages.yml

```diff
@@ -4464,6 +4464,17 @@ POV-Ray SDL:
   tm_scope: source.pov-ray sdl
   ace_mode: text
   language_id: 275
+Pact:
+  type: programming
+  color: "#F7A8B8"
+  ace_mode: text
+  tm_scope: source.pact
+  extensions:
+  - ".pact"
+  - ".repl"
```

Comment:
We need a sample for all extensions being added.

Reply:
Done